servers/src/mining/stratumserver.rs

```diff
@@ -445,12 +449,18 @@ impl StratumServer {
 				return Err(serde_json::to_value(e).unwrap());
 			}
 		};
-
-		let mut b: Block;
+		let mut b: Block = Block::default();
```

Comment:
Im not really happy about creating a throwaway block here.  Is it a performance concern?  I guess it would be better to use Option<Block> and then return invalid share error for None.  I'll make that change unless somebody has a better idea.